src/pages/Targeted.vue

```diff
@@ -20,19 +22,11 @@ export default defineComponent({
   }
 })
 </script>
-
 <style lang="scss" module>
-.container {
-  width: 100%;
-  max-width: fit-content;
-  margin: 1em 0;
+.frame {
   border: solid 1.5px #d9d9d9;
   border-collapse: collapse;
   box-shadow: 0 2px 3px #dfe0d7;
-}
-.header {
-  text-align: left;
-  font-size: 15pt;
-  margin: 8px 0;
+  padding: 2rem;
```

Comment:
```suggestion
  padding: 1rem 1.5rem;
```
こっちの方が見栄え良い気がするのでこのへんわちゃわちゃして申し訳ないんですがこれでお願いします:pray: